dc-model-jackson/src/test/java/de/digitalcollections/model/jackson/identifiable/entity/relation/EntityToFileResourceRelationTest.java

```diff
@@ -0,0 +1,34 @@
+package de.digitalcollections.model.jackson.identifiable.entity.relation;
+
+import de.digitalcollections.model.identifiable.entity.Article;
+import de.digitalcollections.model.identifiable.entity.relation.EntityToFileResourceRelation;
+import de.digitalcollections.model.identifiable.resource.ApplicationFileResource;
+import de.digitalcollections.model.jackson.BaseJsonSerializationTest;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+
+public class EntityToFileResourceRelationTest extends BaseJsonSerializationTest {
```

Comment:
Hier bitte noch die `@DisplayName` -Annotation nutzen, beispielsweise `@DisplayName("The EntityToFileResourceRelation`)

Reply:
ist nachgezogen